.github/workflows/dco.yml

```diff
@@ -0,0 +1,19 @@
+name: DCO Check
```

Comment:
I think it might need to be called DCO to satisfy the GH requirement check?

Reply:
I will update the name.

Reply:
Updated